### PR TITLE
Feature/processor networkmessage

### PR DESCRIPTION
**Pull request by cristipogacean**
* added PubSub json and uadp handling in the telemetry processors
* added Display Name processing in the publisher

### CODE REVIEW

File: common/src/Microsoft.Azure.IIoT.Core/src/Messaging/EventHub/Services/EventHubDeviceEventHandler.cs [hunk 1]
```diff
@@ -52,7 +56,7 @@ public sealed class EventHubDeviceEventHandler : IEventHandler {
         /// <inheritdoc/>
         public async Task OnBatchCompleteAsync() {
             foreach (var handler in _used) {
-                await Try.Async(_handlers[handler].OnBatchCompleteAsync);
+                await Try.Async(_handlers[handler]!.OnBatchCompleteAsync);
```
Comment (marcschier) [inline common/src/Microsoft.Azure.IIoT.Core/src/Messaging/EventHub/Services/EventHubDeviceEventHandler.cs]: Is this intentended?

File: common/src/Microsoft.Azure.IIoT.Core/src/Utils/JsonConvertEx.cs [hunk 2]
```diff
@@ -57,7 +57,11 @@ public static class JsonConvertEx {
         /// <returns></returns>
         internal static JsonSerializerSettings GetDefaultSettings() {
             return new JsonSerializerSettings {
-                ContractResolver = new DefaultContractResolver(),
+                ContractResolver = new DefaultContractResolver() {
+                    NamingStrategy = new CamelCaseNamingStrategy {
+                        ProcessDictionaryKeys = false
```
Comment (marcschier) [inline common/src/Microsoft.Azure.IIoT.Core/src/Utils/JsonConvertEx.cs]: QQ: where is this coming into play?

File: common/src/Microsoft.Azure.IIoT.Core/src/Messaging/EventHub/Services/EventHubDeviceEventHandler.cs (hunk 1 above)
Comment (cristipogacean) [reply common/src/Microsoft.Azure.IIoT.Core/src/Messaging/EventHub/Services/EventHubDeviceEventHandler.cs]: It was an attempt to get rid of the NullReferenceException we had, I will revert it.

File: common/src/Microsoft.Azure.IIoT.Core/src/Utils/JsonConvertEx.cs (hunk 2 above)
Comment (cristipogacean) [reply common/src/Microsoft.Azure.IIoT.Core/src/Utils/JsonConvertEx.cs]: this is required for the serialization of the payload dictionary if form "mykey": {"value":"myvalue"}

Comment (marcschier) [reply common/src/Microsoft.Azure.IIoT.Core/src/Utils/JsonConvertEx.cs]: Got it, thanks!

**Pull request merged**